src/ProjectWindow.cpp

```diff
@@ -1880,11 +1893,32 @@ void ProjectWindow::DoZoomFit()
    window.TP_ScrollWindow(start);
 }
 
-void ProjectWindow::ShowEffectsPanel(Track* track)
+void ProjectWindow::ShowEffectsPanel(AudacityProject& project, Track* track)
 {
+   mEffectsTrackSelectionChanged.Reset();
+
    if(track == nullptr)
       return;
 
+   wxWindowUpdateLocker freeze(this);
+
+   mEffectsTrackSelectionChanged = 
+      track->GetOwner()->Subscribe([this](const TrackListEvent &event)
```

Comment:
This still subscribes to selection change efents, but we have agreed that is should instead be focus changes.

Also I suggest that the effects window might subscribe itself rather than relying on `ProjectWindow` to make the changes.

Reply:
When the button is pressed to change the track whose effect list is shown -- then the track focus should also be changed to that track.

Reply:
Hm, seems like I've lost part of my changes during commit history rewrite